src/startupscriptgenerator/src/python/scriptgenerator.go

```diff
@@ -9,85 +9,103 @@ import (
 	"common"
 	"common/consts"
 	"fmt"
-	"io/ioutil"
 	"os"
 	"path/filepath"
 	"runtime"
 	"strconv"
 	"strings"
 )
 
+
 type PythonStartupScriptGenerator struct {
-	SourcePath               string
-	UserStartupCommand       string
-	DefaultAppPath           string
-	DefaultAppModule         string
-	BindPort                 string
-	VirtualEnvironmentName   string
-	PackageDirectory         string
-	SkipVirtualEnvExtraction bool
-	Manifest                 common.BuildManifest
+	AppPath                     string
+	UserStartupCommand          string
+	DefaultAppPath              string
+	DefaultAppModule            string
+	DefaultAppDebugModule       string
+	DebugAdapter                string // Remote debugger adapter to use. Currently, only `ptvsd` is supported.
```

Comment:
we can just remove `DebugAdapter` since only one is supported

Reply:
I had future-proofing in mind - we might want to support PyCharm for example